src/sentry/http.py

```diff
@@ -265,3 +266,32 @@ def fetch_file(
             response.close()
 
     return UrlResult(url, result[0], result[1], result[2], result[3])
+
+
+def record_http_metrics(response, key=None, instance=None, tags=None):
+    key = key or 'request'
+    tags = tags or {}
+    instance = instance or 'sentry.http'
+
+    metrics.incr(
+        u'{}.sent'.format(key),
+        instance=instance,
+        tags=tags,
+        skip_internal=False,
```

Comment:
Do you need *all* the metric values? Or are you ok with the default sampling rates?

Reply:
Hmm, good question. I don't know if all of the events will have enough volume to warrant sampling, but some will (eg. `error.created`).